pom.xml

```diff
@@ -735,6 +742,9 @@
         <javaee.web.api.version>7.0</javaee.web.api.version>
         <h2database.version>1.0.60</h2database.version>
 
+        <commons-codec.test.version>1.4</commons-codec.test.version>
+
+
```

Comment:
Remove the extra line

Reply:
fixed in latest: https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/09dcf9f2ddc3465f747118331cf1f6c7a0ad8f14/pom.xml#L956